Pod/Classes/NSURLSession/NSURLSession+Mock.swift

```diff
@@ -134,14 +141,24 @@ extension NSURLSession {
     public class func removeAllMocks(of request: NSURLRequest) {
         self.register.removeAllMocks(of: request)
     }
+    
+    /**
+     For a given handle, has it been consumed or not?
+     
+     - paramater handle: The returned handle from a call to mock a request i.e. mockSingle(...)
+     */
+    public class func hasMockConsumed(handle: Handle) -> Bool {
```

Comment:
Good point - and actually fairly tricky to fix, we don't store a list of ephemeral mocks which have been consumed. How important is this to fix?